### PR TITLE
Add PolyLineT ctor that takes a temporary std::vector

### PR DESCRIPTION
**Pull request by MikeGitb**
Nothing particularly important, but it came up in a program of mine

### CODE REVIEW

File: src/cinder/FileWatcher.cpp [hunk 1]
```diff
@@ -45,7 +45,7 @@ class Watch : public std::enable_shared_from_this<Watch>, private Noncopyable {
 	void checkCurrent();
 	//! Remove any watches for \a filePath. If it is the last file associated with this Watch, discard
 	void unwatch( const fs::path &filePath );
-	//! Emit the signal callback. 
+	//! Emit the signal callback.
```
Comment (drewish) [inline src/cinder/FileWatcher.cpp]: I'm all for whitespace cleanup but this other file seems out of scope for the PR

**Comment by MikeGitb** [comment]
@drewish: Sorry, you are completely right - don't know how that slippe through.

**Comment by MikeGitb** [comment]
FYI: The ci failure is because the mac tasks were canceled (for whatever reason) not due to an actual error.

**Comment by richardeakin** [comment]
Thanks!
